.gitignore

```diff
@@ -1,12 +1,19 @@
 ### Unwanted Terraform files when using the examples ###
-
 examples/*/.terraform
 examples/*/terraform.tfstate
 examples/*/terraform.tfstate.*
 examples/*/.terraform.tfstate.lock.info
 examples/*/*-config
 examples/*/.terraform.lock.hcl
-
+examples/*/secrets.yml
```

Comment:
This should no be here, because we add these files with a placeholder content to the examples. See [here](https://github.com/camptocamp/devops-stack/blob/example-scaleway/examples/eks/secrets.yml), for example.